cms/management/commands/import_from_git.py

```diff
@@ -59,6 +65,39 @@ def emit(self, message):
         if not self.quiet:
             self.stdout.write(message)
 
+    def get_thumbor_image_file(self, host, uuid, use_safe_url=False):
+        '''
+        NOTE: To download an image from the Thumbor host we
+        either need a security key for the particular host,
+        or the host must allow unsafe urls.
+        '''
+        if use_safe_url:
+            url = thumbor_image_url(uuid)
+        else:
+            url = urljoin(host, 'unsafe/%s' % uuid)
```

Comment:
`unsafe` is currently disabled. I'd say use `/image/{uuid}` - it retrieves the raw image that was uploaded.